app/services/create_user_service.rb

```diff
@@ -1,23 +1,17 @@
 class CreateUserService
   def self.invite_responsible_body_user(user_params = {})
-    user = User.new(user_params.merge(override_params))
+    user = User.new(user_params)
     if user.save
       InviteResponsibleBodyUserMailer.with(user: user).invite_user_email.deliver_later
     end
     user
   end
 
   def self.invite_school_user(user_params = {})
-    user = User.new(user_params.merge(override_params))
+    user = User.new(user_params)
```

Comment:
I think you've found a bug here... it looks like previously, even if a new school user was invited without a TechSource account, one was still being created for them. I'd suggest we fix the bug as part of this PR.

So for `self.invite_school_user` I think we want to let outside callers to set `orders_devices` but not `approved_at`.

Reply:
Made a tweak, should now work as you've described